.travis/run_travis_job.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+# Fail immediately in case of errors and/or unset variables
+set -eu -o pipefail
+
+# Echo commands so that the progress can be seen in CI server logs.
+set -x
+
+# Install clippy
+if rustc --version | grep -q "nightly"
+then
+    # Install nightly clippy
+    rustup component add clippy --toolchain=nightly || cargo install --git https://github.com/rust-lang/rust-clippy/ --force clippy
+else
+    # Install stable clippy
+    rustup component add clippy
+fi
+cargo clippy -V
+
+# Install rustfmt
+rustup component add rustfmt
+rustfmt -V
+
+echo 'Performing checks over the rust code'
+# Check the formatting.
+cargo fmt --all -- --check
+
+# Detect installed Java
+export JAVA_HOME="$(java -XshowSettings:properties -version \
```

Comment:
Let's move it after clippy, just to keep it dense.

Reply:
It is intentionally before `clippy` because `clippy` executes `build.rs` checks for `JAVA_HOME` or tries to detect it from installed java if `JAVA_HOME` is not defined. We need `JAVA_HOME` anyways to construct `LD_LIBRARY_PATH` which is required for integration tests, so maybe it would be better to just put this section on top of the script?